.github/labeler.yml

```diff
@@ -0,0 +1,17 @@
+schedule:
+- any: ['src/components/SchedulePage/**/*', 'src/pages/schedule.js']
```

Comment:
This probably doesn't do what you want it to do. https://github.com/actions/labeler#match-object says that `any` means

> match ALL globs against ANY changed path.

It's obviously impossible for any changed path to match both of these globs.

Instead, you should do this:

```yaml
schedule:
- src/components/SchedulePage/**/*
- src/pages/schedule.js
```

Same goes for everything else.